### ISSUE TITLE
[Feature Request][SF][1.28.4][List][A11y][Expected Fixed Time: ASAP] Cannot change the title heading level

### ISSUE THREAD

**Issue by wilson-xu-sap** [SAP/ui5-webcomponents-react#6180]
### Is your feature request related to a problem?

Cannot change title heading level when use titleText, which violates the a11y standard.

### Describe the solution you'd like

Add a new prop for setting and calculating the heading level.

https://github.com/SAP/ui5-webcomponents/issues/9652

### Organization

SF

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
I will close this issue as there is already[ another one](https://github.com/SAP/ui5-webcomponents/issues/9652) covering this topic. If you have any questions or more information, please address them in the original issue.

Duplicate https://github.com/SAP/ui5-webcomponents/issues/9652
